jolt-core/src/jolt/vm/ram.rs

```diff
@@ -315,7 +327,7 @@ impl<F: JoltField, ProofTranscript: Transcript> BatchableSumcheckInstance<F, Pro
     for BooleanitySumcheck<F>
 {
     fn degree(&self) -> usize {
-        3
+        self.d + 2
```

Comment:
I missed this earlier; this should still be 3 (we're taking a random linear combination of d degree-3 sumchecks)